lib/eventmesh/EventMeshServer.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const errors = require('../errors');
+const NotImplementedBySubclass = errors.NotImplementedBySubclass;
+
+class EventMeshServer {
+  constructor() {}
+
+  registerService(resourceType, serviceId, serviceAttributesValue, servicePlansValue) {
+    /* jshint unused:false */
+    throw new NotImplementedBySubclass('registerService');
+  }
+
+  getServiceAttributes(resourceType, serviceId) {
```

Comment:
What is this used for?

Reply:
To get the object of service details common to all plans 
like service name , service guid etc.